tree-construction/tests20.dat

```diff
@@ -514,3 +514,57 @@
 |       <math annotation-xml>
 |         encoding=" text/html "
 |     <div>
+
+#data
+<math><annotation-xml> </annotation-xml>
+#errors
```

Comment:
@nolanw I just don't have anything off-hand that's up-to-date to generate them from, which is what makes it a real pain. (HTMLReader doesn't have any easy-to-run test binary, does it? That, say, takes input from either a file or stdin and dumps the test expectation format?)